src/components/Settings.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
Please do not turn off warnings if you can fix them. This warning is very reasonable.
```suggestion
```